app/src/main/java/app/gamenative/ui/component/ScreenEffectsPanel.kt

```
@@ -433,7 +457,49 @@ fun GLScreenEffectsTabContent(
+                    valueText = formatPercent(barrelDistortionStrength),
+                    progress = normalizedProgress(barrelDistortionStrength, 0f, 100f),
+                    onDecrease = {
+                        barrelDistortionStrength = (barrelDistortionStrength - SCREEN_EFFECT_PERCENT_STEP).coerceIn(0f, 100f)
+                    },
+                    onIncrease = {
+                        barrelDistortionStrength = (barrelDistortionStrength + SCREEN_EFFECT_PERCENT_STEP).coerceIn(0f, 100f)
+                    },
+                )
+                ScreenEffectAdjustmentRow(
+                    title = stringResource(R.string.screen_effects_barrel_distortion_fov),
+                    valueText = String.format("%.0f°", barrelDistortionFov),
```

detekt (1.23.8)

[warning] String.format("%.0f°", barrelDistortionFov) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)